ui/app/templates/components/mount-backend-form.hbs

```diff
@@ -21,7 +21,7 @@
 <form {{action (perform this.mountBackend) on="submit"}}>
   <div class="box is-sideless is-fullwidth is-marginless">
     <NamespaceReminder @mode="enable" @noun={{if (eq this.mountType "auth") "Auth Method" "Secret Engine"}} />
-    <MessageError @model={{this.model}} @errorMessage={{this.errorMessage}} />
```

Comment:
is there a way to get the model to update when an AdapterError happens in the controller?

Reply:
I think I need more clarification. We can sync offline